test/gtest/reducecalculation.cpp

```diff
@@ -61,6 +71,19 @@ TEST_P(GPU_ReduceCalculationTest_BFP16, ReduceCalculationTestFw)
     Verify();
 };
 
+TEST_P(GPU_ReduceCalculationTest_INT8, ReduceCalculationTestFw)
+{
+    RunTest();
+    Verify();
+};
+
+TEST_P(GPU_ReduceCalculationTest_UINT8, ReduceCalculationTestFw)
+{
+    RunTest();
+    Verify();
```

Comment:
```suggestion
TEST_P(GPU_ReduceCalculationTest_INT8, Test)
{
    RunTest();
    Verify();
};

TEST_P(GPU_ReduceCalculationTest_UINT8, Test)
{
    RunTest();
    Verify();
```
naming convention https://github.com/ROCm/MIOpen/wiki/GTest-development

Reply:
solved